### PR TITLE
[YUNIKORN2729] remove `--new-from-rev` from Makefile

### PR DESCRIPTION
**Pull request by rich7420**
### What is this PR for?
- fix lint error

### What type of PR is it?
 - Improvement

### Todos
remove `--new-from-rev` from Makefile in core

### What is the Jira issue?
[YUNIKORN-2729](https://issues.apache.org/jira/browse/YUNIKORN-2729)

### How should this be tested?
remove `--new-from-rev` from Makefile 
`make lint`

### Questions:
N/A

### CODE REVIEW

File: pkg/client/kubeclient.go [hunk 1]
```diff
@@ -138,7 +138,6 @@ func (nc SchedulerKubeClient) Create(pod *v1.Pod) (*v1.Pod, error) {
 }
 
 func (nc SchedulerKubeClient) Delete(pod *v1.Pod) error {
-	// TODO make this configurable for pods
```
Comment (chia7712) [inline pkg/client/kubeclient.go:141]: Could you please move this TODO to jira?

File: test/e2e/framework/helpers/common/utils.go [hunk 2]
```diff
@@ -102,7 +103,11 @@ func RandSeq(n int) string {
 	var letters = []rune("abcdefghijklmnopqrstuvwxyz0123456789")
 	b := make([]rune, n)
 	for i := range b {
-		b[i] = letters[rand.Intn(len(letters))]
```
Comment (chia7712) [inline test/e2e/framework/helpers/common/utils.go:105]: please add the ignore as we don't need crypt quality

**Review by chia7712 — COMMENTED** [review]
@rich7420 thanks for this patch

File: pkg/client/kubeclient.go (hunk 1 above)
Comment (rich7420) [reply pkg/client/kubeclient.go:141]: Oh yes, I had reported a jira. https://issues.apache.org/jira/browse/YUNIKORN-2734

File: test/e2e/framework/helpers/common/utils.go (hunk 2 above)
Comment (rich7420) [reply test/e2e/framework/helpers/common/utils.go:105]: No problem.

**Review by chia7712 — APPROVED** [review]
LGTM

File: Makefile [hunk 3]
```diff
@@ -291,7 +291,7 @@ lint: $(GOLANGCI_LINT_BIN)
 	@git symbolic-ref -q HEAD && REV="origin/HEAD" || REV="HEAD^" ; \
 	headSHA=$$(git rev-parse --short=12 $${REV}) ; \
 	echo "checking against commit sha $${headSHA}" ; \
-	"${GOLANGCI_LINT_BIN}" run --new-from-rev=$${headSHA}
+	"${GOLANGCI_LINT_BIN}"
```
Comment (chia7712) [inline Makefile:294]: please keep `run` ...

**Pull request closed without merging**